sim/paladin/judgement.go

```diff
@@ -45,33 +47,99 @@ func (paladin *Paladin) registerJudgement() {
 			//}
 			// paladin.currentSeal.Deactivate(sim)
 
+			thisJudgement := 0 // Seals Judged this Judgement
+
 			// Phase 4 - (Not tied to T1 6pc bonus) - Judge all Seals (2 possible without 6pc, 2 or 3 with 6pc TBD)
 			for _, sealAura := range paladin.aurasSoC {
 				if sealAura.IsActive() {
 					spell.CalcAndDealOutcome(sim, target, spell.OutcomeMagicHit)
-					sealAura.Deactivate(sim)
+					if !paladin.t2Judgement2pc {
```

Comment:
To make this a bit more extendable in case this effect ever comes back, this attribute could be something like `consumeSealsOnJudge`

Reply:
Another option is just tracking `SpellCode`

Reply:
added consumeSealsOnJudge